library/modules/Units.cpp

```diff
@@ -541,6 +541,23 @@ string Units::getRaceName(df::unit* unit)
     return getRaceNameById(unit->race);
 }
 
+void df_unit_physical_description(df::unit* unit, string* out_str)
+{
+    static auto* const fn =
+        reinterpret_cast<void(THISCALL *)(df::unit*, string*)>(
+            Core::getInstance().vinfo->getAddress("unit_physical_description"));
+    CHECK_NULL_POINTER(fn);
```

Comment:
I would actually keep the do-nothing behavior that you had earlier if fn is null - when global addresses are missing, most other functions also fall back to some default behavior. Setting out_str to an empty string is also an option.

Reply:
done 👍